[DOCS] Adds #112562 known issue to 7.14.2 release notes (#112744)
elastic/kibana: docs/CHANGELOG.asciidoc

Review the following information about the 7.14.2 release.

[float]
[[known-issue-v7.14.2]]
=== Known issue

> {kib} is unable to restore *Discover* search sessions with a relative time range. When you restore the *Discover* search session, then run a new searche, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430].

Reviewer: ```suggestion
{kib} is unable to restore *Discover* search sessions with a relative time range. When you restore the *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430].
```

Reviewer 2: ```suggestion
{kib} is unable to restore *Discover* search sessions with a relative time range. When you restore the *Discover* search session, {kib} starts a new search and displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430].
```

I think it should be something like this
